tox.ini

```diff
@@ -19,6 +19,7 @@ commands =
 
 [flake8]
 exclude = .tox,*.egg
+max-complexity = 9
```

Comment:
Most of the codebase is actually only complexity 5, it's only `validation_tween_factory` that requires a 9 I think.